WordPress/src/main/java/org/wordpress/android/util/config/BloganuaryNudgeFeatureConfig.kt

```diff
@@ -0,0 +1,20 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.Feature
+import javax.inject.Inject
+
+private const val BLOGANUARY_NUDGE_REMOTE_FIELD = "bloganuary_dashboard_nudge"
+
+@Feature(BLOGANUARY_NUDGE_REMOTE_FIELD, false)
```

Comment:
💡 Would it make sense to have the default value as `true` in case this is being merged directly in the new version branch?